wandb/sdk/wandb_run.py

```diff
@@ -2551,7 +2551,10 @@ def _on_ready(self) -> None:
         self._telemetry_obj_active = True
         self._telemetry_flush()
 
-        self._detect_and_apply_job_inputs()
+        try:
+            self._detect_and_apply_job_inputs()
```

Comment:
do you understand why it fails in the way it does?

Reply:
not enough info in the sentry trace to be sure, but I'd guess something about the import path / screwed env setup. But it doesn't matter anymore, no?